cms/src/test/java/uk/nhs/digital/apispecs/swagger/request/examplerenderer/CodegenParameterExampleHtmlRendererTest.java

```diff
@@ -40,7 +40,8 @@ public void rendersExampleSpecifiedInParamDefinition_evenWhenExampleIsSpecifiedI
 
         // given
         final String expectedHtmlForExampleValue =
-            "Example: <span class=\"nhsd-a-text-highlight nhsd-a-text-highlight--code\">simple example specified in parameter's definition</span>";
+            "Example: <span class=\"nhsd-a-text-highlight nhsd-a-text-highlight--code\">"
```

Comment:
if only java had a nice syntax for multiline strings!

Reply:
Indeed. If we weren't stuck on Java 8, [we would have it](https://www.baeldung.com/java-multiline-string) :grin: